contrib/labstack/echo.v4/echotrace.go

```diff
@@ -47,7 +49,15 @@ func Middleware(opts ...Option) echo.MiddlewareFunc {
 			defer span.Finish()
 
 			// pass the span through the request context
-			c.SetRequest(request.WithContext(ctx))
+			req := request.WithContext(ctx)
+			c.SetRequest(req)
+
+			if appsec.Enabled() {
+				op := httpsec.StartOperation(httpsec.MakeHandlerOperationArgs(req, span), nil)
+				defer func() {
+					op.Finish(httpsec.HandlerOperationRes{Status: c.Response().Status})
+				}()
+			}
 
```

Comment:
Can you please remove this empty line?

Reply:
Fixed in https://github.com/DataDog/dd-trace-go/pull/1042/commits/89a0faab579118328372ae6761d324707ccbad3c